app/controllers/emails_controller.rb

```diff
@@ -23,6 +24,11 @@ def submit
     end
   end
 
+  def submit_generic
+    @send_generic = true
```

Comment:
So that the template knows whether it should redirect to the submit or the submit_generic action. Wasn't necessary initially but they changed the email interface within the past weeks to make it far more specific then it was originally. Could use 2 different views instead, but thought it would be more convenient to just pass a parameter this way. I dont know whether there might be an better way (one could think of sending an path in there?)